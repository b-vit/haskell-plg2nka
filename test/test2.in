A,B
a,b,c
A
