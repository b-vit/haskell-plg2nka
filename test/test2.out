flp21-fun: Input the grammar in format: 
 Nonterminals 
 Terminals
 Starting Symbol
 Production Rules
 for example:
 A,B
 a,b
 A
 A->B
 B->A
 ...
 B->#
CallStack (from HasCallStack):
  error, called at src/Parser.hs:19:5 in main:Parser